find-closest-facility-from-point/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <string name="app_name">Find closest facility to an incident (interactive)</string>
```

Comment:
This the the old sample name, please update this to: `Find closest facility from point`